controllers/storagecluster_controller.go

```diff
@@ -45,17 +58,111 @@ type StorageClusterReconciler struct {
 // For more details, check Reconcile and its Result here:
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.8.3/pkg/reconcile
 func (r *StorageClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	_ = log.FromContext(ctx)
+	logger := log.FromContext(ctx)
+
+	instance := &ocsv1.StorageCluster{}
+	err := r.Client.Get(context.TODO(), req.NamespacedName, instance)
+	if err != nil {
+		if errors.IsNotFound(err) {
+			logger.Info("storagecluster instance not found")
+			return ctrl.Result{}, nil
+		}
+		// Error reading the object - requeue the request.
+		return ctrl.Result{}, err
+	}
+
+	logger.Info("storagecluster instance found")
+
+	// get list of StorageSystems
+	storageSystemList := &odfv1alpha1.StorageSystemList{}
+	err = r.Client.List(context.TODO(), storageSystemList, &client.ListOptions{Namespace: instance.Namespace})
+	if err != nil {
+		logger.Error(err, "failed to list the storageSystem")
```

Comment:
We should start trying to adhere to better logging standards. Make sure all messages start with a capital letter and end with a period.

Reply:
Done